cross-functional/workflow.md

```diff
@@ -113,19 +113,19 @@ There are two delivery boards:
 
 ### Operations board
 
-The [**Operations board**](https://github.com/orgs/2i2c-org/projects/50) is a team-wide board focused around delivery. Includes all organizational tasks/actions except for Engineering.
+The [**Operations board**](https://github.com/orgs/2i2c-org/projects/50) is a board focused on delivering operational initiatives, like building and improving processes. It's owned by our Delivery Enablement team and supports all other members of the organization. This isn’t a “just catch all the other things board"; everything flows from our strategic goals and initiatives. 
 
 - [**Board link**](https://github.com/orgs/2i2c-org/projects/50)
 - [**`#initiatives`**](https://2i2c.slack.com/archives/C06G5FAAT63) is the Slack channel for this board.
-- **Meetings**: Bi-weekly organization-wide sprint planning meetings.
+- **Meetings**: Fortnightly planning, monthly retrospective, and weekly coordination.
 
```

Comment:
```suggestion
- **Meetings**:
    - Weekly coordination
    - Every-two-weeks planning
    - Monthly retrospective

```

Reply:
I found it hard to figure out if this was referring to one meeting or many, does this help?